src/mudlet-lua/lua/Other.lua

```diff
@@ -3,7 +3,7 @@
 ----------------------------------------------------------------------------------
 
 mudlet = mudlet or {}
-mudlet.supportscoroutines
+mudlet.supportscoroutines = true
```

Comment:
`mudlet.supportscoroutines` will error out anyways, because the `mudlet` table already resolves to `nil`. Maybe we should go with `mudlet.supports.*`?